packages/renderer/src/lib/ui/EmptyScreen.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faPaste } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
+
+export let title: string = 'No title';
+export let message: string = 'Message';
+export let commandline: string = '';
+export let hidden: boolean = false;
+
+function copyRunInstructionToClipboard() {
+  const text = copyText?.innerText;
+  navigator.clipboard.writeText(text);
+}
+
+let copyText;
+
+onMount(async () => {
+  copyText = document.getElementById('CommandLine') as HTMLElement;
+});
```

Comment:
we may have multiple commandLine elements on a page now that you're reusing the same component

so I would switch from document:getElementById('id') to `bind:this="{myComponent}"` form

declare variable
```ts
let copyTextDivElement: HTMLDivElement;
```

assign variable from the html code
```html
<div bind:this="{copyTextDivElement}">{commandline}</div>
```

Reply:
Fixed in new commit.